gamemode/sv_mapvote.lua

```diff
@@ -42,9 +42,7 @@ function GM:ChangeMapTo(map)
 		return
 	end
 	print("[husklesph] Rotate changing map to " .. map)
-	local ct = ChatText()
-	ct:Add("Changing map to " .. map)
-	ct:SendAll()
+	GAMEMODE:ChatMsg("Changing map to ", map)
```

Comment:
Is there a reason different files inconsistently use `GAMEMODE:ChatMsg` versus `self:ChatMsg`?

Reply:
There are two different ChatMsg functions. One of them is part of PlayerMeta and the other is in  GameMode. The one for PlayerMeta is used to send a message to a particular player and the one in GameMode sends a message to all players. I made the usage consistent such that if sending a message to all players then GAMEMODE.ChatMsg is used. The other variations are for sending a message to a particular player.

However I see the point you were (maybe) making and it's vague which version is being used since they are named the same. It would be a good idea to more clearly name each version.

Reply:
I have renamed both ChatMsg methods and given them more clear names. All references to these methods have also been updated to reflect the new names. It is now much clearer which one is being used.